internal/command/commands/env.go

```diff
@@ -205,81 +153,39 @@ func (e *Env) Run(args []string) int {
 			return 1
 		}
 
-		tableHeaders := []string{"Name", "Team", "Env Type", "State", "Account", "Deletion Time", "Purpose", "CreatedAt", "CreatedBy", "UpdatedAt", "UpdatedBy"}
+		tableHeaders := []string{"Name", "Team", "Env Type", "State", "Account", "Deletion Time"}
```

Comment:
Why createdBy, updatedBy etc are removed.
Can't find in the doc

Reply:
In a discussion with Rahul, requirement was to reduce the size of table to simplify display. So, we decided to trim the table by removing these cols.

Reply:
I think we should atleast have createdBy column

Reply:
This is not decided yet. Will modify if required in later stages depending upon discussion.